test/changeType.test.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert');
```

Comment:
I don't think this test file is necessary in my opinion

if it's the dictionary, it exists, right?